src/ui/base-page-storyEdit.js

```diff
@@ -252,28 +263,26 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
         $(pageContainer).toggleClass(hiddenEditorClass, !authoringEnabled);
     };
 
-    sjrk.storyTelling.base.page.storyEdit.submitStory = function (that, errorEvent) {
-        var form = that.container.find("form");
-
-        form.attr("action", "/stories/");
-        form.attr("method", "post");
-        form.attr("enctype", "multipart/form-data");
+    sjrk.storyTelling.base.page.storyEdit.submitStory = function (storyEditorForm, storyModel, errorEvent) {
+        storyEditorForm.attr("action", "/stories/");
+        storyEditorForm.attr("method", "post");
+        storyEditorForm.attr("enctype", "multipart/form-data");
```

Comment:
You can do this in a single `attr` call by using an object as the argument. e.g. 

```json
{
    "action": "/stories/",
    "method": "post",
    "enctype": "multipart/form-data"
}
```
see: https://api.jquery.com/attr/#attr-attributes